src/happy_vllm/core/resources.py

```diff
@@ -46,8 +47,13 @@ async def lifespan(app: FastAPI):
         logger.info("Model loaded")
 
         RESOURCES[RESOURCE_MODEL] = model
+
+        # Load the tools
+        update_tools(args=args)
+
```

Comment:
Sure, the implementation doesn't match with happy_vllm ideas, so :
- Move to example directory to keep the implementation because be useful for someone to need deploy on local
- We could create a new route which could update the tool_choice and tools variables (avoid to use in production)